openedx_events/tests/utils.py

```diff
@@ -124,3 +127,32 @@ def start_events_isolation(cls):
         cls().disable_all_events()
         cls().enable_events_by_type(*cls.ENABLED_OPENEDX_EVENTS)
         cls().allow_send_events_failure(*cls.ENABLED_OPENEDX_EVENTS)
+
+
+def load_all_signals():
+    """
+    Ensure OpenEdxPublicSignal.all_events() cache is fully populated.
+
+    Loads all non-test signals.py modules.
+    """
+    found = set()
+
+    root = import_module('openedx_events')
+    for m in pkgutil.walk_packages(root.__path__, root.__name__ + '.'):
+        module_name = m.name
+        if 'tests' in module_name.split('.') or '.test_' in module_name:
+            continue
```

Comment:
I don't actually care about coverage here, but it's _weird_ that this line is allegedly not covered. There should be about 10 test modules that hit this line during the walk, and I see them locally if I put a print statement here!

Reply:
This is because Python has optimized away that line: https://github.com/nedbat/coveragepy/issues/198